### ISSUE TITLE
SPin8 -> SPan8 DA settings

### ISSUE THREAD

**Issue by 1190202328** [jamycheung/Trans4PASS#15]
Hi there, may I ask if you can provide the configuration file for SPin8->Span8 DA settings? In this repository, train_warm.py, train_ssl.py, and train_mpa.py are the codes for the indoor DA settings. I would like to know the hyper-parameters of the outdoor DA settings, such as learning rate and optimizer. Because I am currently unable to reproduce the 67.16 mIoU of SPin8->Span8 in the Trans4PASS+ paper at TABLE 12.(b). Thanks!
